README.md

```diff
@@ -31,10 +46,14 @@ module "iam_user" {
 
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-------:|:--------:|
-| ssm_parameters | The AWS SSM parameters that the IAM user needs to be able to read | list(string) | | yes |
-| user_name | The name to associate with the AWS IAM user (e.g. test-ansible-role-cyhy-core) | string | | yes |
 | add_packer_permissions | Whether or not to give the IAM user the permissions needed by packer to create an AMI | bool | `false` | no |
+| ec2amicreate_role_name | The name of the IAM role in the Images account that allows all of the EC2 actions needed to create an AMI. | string | `EC2AMICreate` | no |
+| images_account_id | The AWS account ID containing the SSM parameter store that the IAM user needs to read from and also where the user will be allowed to create images (if add_packer_permissions is true); if not provided, the current calling account ID is used | string | ID of calling account | no |
+| parameterstorereadonly_role_description | The description to associate with the IAM role (as well as the corresponding policy) that allows read-only access to the specified SSM Parameter Store parameters.  Note that a \"%s\" in this value will get replaced with the user_name variable. | string | `Allows read-only access to SSM Parameter Store parameters required for %s.` | no |
```

Comment:
You don't need the backslashes here.  Only in `variables.tf`.

Reply:
Great catch - fixed in https://github.com/cisagov/molecule-packer-ci-iam-user-tf-module/pull/8/commits/11770729a3a239e6eb33b58997a80e81abbec419